lib/impure/nre.nim

```diff
@@ -495,8 +492,12 @@ proc matchImpl(str: string, pattern: Regex, start, endpos: int, flags: int): Opt
 
 proc match*(str: string, pattern: Regex, start = 0, endpos = int.high): Option[RegexMatch] =
   ## Like ```find(...)`` <#proc-find>`_, but anchored to the start of the
-  ## string. This means that ``"foo".match(re"f") == true``, but
-  ## ``"foo".match(re"o") == false``.
+  ## string.
+  ##
```

Comment:
nit: is this empty `##` line needed? (+ elsewhere)

Reply:
Probably not, but I like the way it looks and separates the example from the rest of the docstring.